charts/kuberpult/values.yaml

```
@@ -162,6 +162,9 @@ db:
     rollout:
       maxOpen: 100
       maxIdle: 20
+  # Kuberpult only checks for custom migrations if this flag is enabled. Otherwise, it skips them entirely, assuming
+  # that there is no information we want to migrate from an existing manifest repository to the database.
```

We should provide clear guidelines on when this flag should be used.
```suggestion
  # that there is no information we want to migrate from an existing manifest repository to the database.
  # You should only enable this flag, if you want to do a one-step upgrade of kuberpult from a (relatively old) version that still uses git as storage.
  # If you are already using the database, ignore this flag.
```